src/promptflow-core/promptflow/_internal/__init__.py

```diff
@@ -0,0 +1,9 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+__path__ = __import__("pkgutil").extend_path(__path__, __name__)  # type: ignore
```

Comment:
why do this? _internal is in devkit package

Reply:
move to devkit package. this function will be called in runtime, they suggest better not used directly in runtime